packages/core/src/ci-environment/index.ts

```diff
@@ -11,13 +12,14 @@ import { debug } from "../debug";
 export { CiEnvironment };
 
 const services = [
-  heroku,
-  githubActions,
-  circleci,
-  travis,
+  bitrise,
   buildkite,
-  gitlab,
+  circleci,
   git,
+  githubActions,
+  gitlab,
+  heroku,
+  travis,
```

Comment:
Please keep the most used top, since we run the detection algo in order, it is better to put the most used first. Git should always be the last.

Reply:
By puting git before others you will break everything.